ClassLibrary/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ClassLibrary
+{
+    public class Employee
+    {
+        private string firstName;
+        private string lastName;
+        private int age;
+        private int experienceYears;
+        private string position = "Intern";
+
+        public string getPosition
```

Comment:
Property should be named with the same word as field, without "get" part

Reply:
fixed